lib/ui/form/widget/form_survey_answer_dropdown.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+
+class FormSurveyAnswerDropdown extends StatefulWidget {
+  const FormSurveyAnswerDropdown({super.key});
+
+  @override
+  State<FormSurveyAnswerDropdown> createState() =>
+      _FormSurveyAnswerDropdownState();
+}
+
+class _FormSurveyAnswerDropdownState extends State<FormSurveyAnswerDropdown> {
+  late List<String> pickerOptions = [];
+
+  int selectedIndex = 1;
+  late FixedExtentScrollController _scrollController;
+
+  @override
+  void initState() {
+    super.initState();
+    _scrollController = FixedExtentScrollController(initialItem: selectedIndex);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    pickerOptions = [
+      AppLocalizations.of(context)!.answer_dropdown_option_very_fullfilled,
+      AppLocalizations.of(context)!.answer_dropdown_option_somewhat_fullfilled,
+      AppLocalizations.of(context)!
+          .answer_dropdown_option_somewhat_unfullfilled,
+    ];
+    return SizedBox(
+      width: AppDimensions.answerDropdownWidth,
+      child: Center(
+        child: CupertinoPicker(
+          scrollController: _scrollController,
+          itemExtent: AppDimensions.answerDropdownItemHeight,
+          onSelectedItemChanged: (index) {
+            setState(() {
+              selectedIndex = index;
+            });
+          },
+          children: List.generate(
+            pickerOptions.length,
+            (index) {
+              final String item = pickerOptions[index];
+              final bool isSelected = selectedIndex == index;
+              final selectedTextStyle = Theme.of(context).textTheme.labelMedium;
+              final notSelectedTextStyle = Theme.of(context)
+                  .textTheme
+                  .labelMedium!
+                  .copyWith(fontWeight: FontWeight.w400);
+              final textStyle =
+                  isSelected ? selectedTextStyle : notSelectedTextStyle;
+
+              return Center(
```

Comment:
Is it possible that this `Center` widget is redundant, since we already define one below? 💭